consensus/src/dag/dag_store.rs

```diff
@@ -46,13 +42,17 @@ pub struct Dag {
     author_to_index: HashMap<Author, usize>,
     storage: Arc<dyn DAGStorage>,
     initial_round: Round,
+    epoch_state: Arc<EpochState>,
+
+    highest_committed_anchor_round: Round,
```

Comment:
I think this needs to be Atomic or Arc<Mutex<>> since callback is invoked from a different thread

Reply:
DagStore is wrapped in a RwLock, right?